### ISSUE TITLE
BUG: The 'Equalizer' Icon Moves Up and Down While playing an Audio Track

### ISSUE THREAD

**Issue by ghost** [naumch1k/one-fall#42]
### Description
This movement creates the effect of a jumping icon. It occurs only in Firefox browser.

### Environment
**Desktop:**
 - OS: Windows 10
 - Browser: Firefox
 - Version: 129.0.1

### Steps to reproduce:
1. Go to  https://develop--one-fall.netlify.app/
2. Click on the **'MUSIC'** menu item.
3. Click on any audio track for playing.
4. Pay attention to the 'Equalizer' icon behavior.

### Actual result:
The 'Equalizer' icon moves up and down while playing an audio track.

### Expected result:
The 'Equalizer' icon dosn't change its location while playing an audio track.

### Attachments:
https://github.com/user-attachments/assets/13c15a32-be5c-42a4-bf30-02cfea701b86

### Additional context
This issue also occurs on mobile device Motorola e7 power, Firefox browser version 129.0 (build #2016035991). It doesn't occur on tablet.

**Comment by naumch1k** [comment]
@Yura33-dev Can you check if you can reproduce this bug? I tested on all devices and browsers I have an access to, including Firefox 129.0.1, but didn't see any issues 🤔

**Comment by Yura33-dev** [comment]
@naumch1k Of course! I have checked this bug on Firefox 129.0.1. I have the same behaviour of Equalizer which @yuliia-harshyna have discovered for this issue. 😨  Equalizer really a little jumping when music playing.

**Comment by naumch1k** [comment]
@Yura33-dev Thanks! So, it probably has something to do with Windows OS since you both see it and I don't  🙂

**UPD:** I just took a closer look at the element, and I do see it on MacOS and Android devices. So,the issue is not with Windows OS but with Firefox.

After further investigation, I noticed that the equalizer stops moving up and down when I remove `transform: translateX(-50%);` from the `.item` element. We might need to consider an alternative approach for centering to resolve this problem.
